### PR TITLE
MII Sample Testing: Add option to dynamically allocate Traefik nodeports ...

### PR DESCRIPTION
**Pull request by tbarnes-us**
… and change MII sample integration test to use this option.

### CODE REVIEW

File: src/integration-tests/model-in-image/deploy-traefik.sh [hunk 1]
```diff
@@ -49,32 +53,60 @@ set +e
...
   cd ${SRCDIR}
 
+  echo "@@ Installing traefik"
+
   # you only need to add the repo once, but we do it every time for simplicity
   helm repo add stable https://kubernetes-charts.storage.googleapis.com/
 
+  set -x
+
   helm install ${TRAEFIK_NAME} stable/traefik \
```
Comment (anpanigr) [inline src/integration-tests/model-in-image/deploy-traefik.sh:86]: Can you use the the script setup.sh  in samples/charts/utils directory to configure traefik.  The latest version of traefik  ( v 2.2.1) has different chart repository ( containous.github.io/traefik-helm-chart), The chart name is not stable/traefik but traefik/traefik.  I am working on a PR to upgrade the traefik version.  All these will be taken care by setup.sh, provided you use the command "setup.sh create traefik" in your script.

Comment (tbarnes-us) [reply src/integration-tests/model-in-image/deploy-traefik.sh:86]: How about I do that in a future PR once the newest version of setup.sh is ready?

Comment (anpanigr) [reply src/integration-tests/model-in-image/deploy-traefik.sh:86]: Note: The interface to setup.sh will not change unless you want a non default Traefik version.

Comment (tbarnes-us) [reply src/integration-tests/model-in-image/deploy-traefik.sh:86]: We will wire in setup.sh in a different pull, as discussed in the meeting.  Please approve if you're fine with this change.

**Comment by tbarnes-us** [comment]
Passes 'ItMiiSample' kind run:  https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/693/

**Pull request merged**